Gopkg.lock

```diff
@@ -326,6 +326,9 @@
     "pkg/client/clientset_generated/clientset/scheme",
```

Comment:
could you put the vendoring commits in its own one?

Reply:
Sure

Reply:
Done